bin/kano-uixinit-systemd

```diff
@@ -126,21 +103,6 @@ kano_keyboard=`python -c "from kano.utils import detect_kano_keyboard; print det
 # loads and sets the keyboard layout
 set_keyboard $kano_keyboard
 
-is_rpi2_or_above=`python -c "from kano.utils.hardware import has_min_performance,\
-RPI_2_B_SCORE ; print has_min_performance(RPI_2_B_SCORE)"`
-
-# By default always go to the Dashboard Mode, except if switching to Desktop Mode,
-# or the Raspberry model is below RPi2B, because Dashboard is not supported.
-if [ "${DESKTOP_MODE}" != "1" ] && [ "$is_rpi2_or_above" == "True" ] ; then
-    logger --id --tag "info" "kano-uixinit terminates into Dashboard Supervisor"
-    # only run updater once each boot
-    [ "$DESKTOP_RUN" -eq 1 ]  &&  sudo kano-updater-quickcheck
```

Comment:
true @Ealdwulf . I have moved `kano-updater-quickcheck` it to the Dashboard supervisor initialization section, because I believe it eventually needs user attention.

 * https://github.com/KanoComputing/os-dashboard/pull/146/commits/b26822f32e586e29d6c42f6a8262ff73494d6cc4